sygnal/sygnal.py

```diff
@@ -20,21 +20,32 @@
 import logging.config
 import os
 import sys
+from typing import Any, Dict, Set, Union, cast
 
+import jaeger_client
```

Comment:
is `jaeger_client` an optional dependency?

Oh, [apparently not](https://github.com/matrix-org/sygnal/blob/d9fb1a3d9d45cdc42435b7a87d1a036471d5ef51/setup.py#L31). Never mind then.